packages/rx_bloc_cli/mason_templates/bricks/lib_auth/__brick__/test/feature_login/view/login_golden_test.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:{{project_name}}/base/models/errors/error_model.dart';
+
+import '../../helpers/golden_helper.dart';
+import '../../helpers/models/scenario.dart';
+import '../factory/login_factory.dart';
+import '../stubs.dart';
+
+void main() {
+  runGoldenTests([
+    generateDeviceBuilder(
+        widget:
+        loginFactory(loggedIn: false, showErrors: false, isLoading: false),
+        //example: Stubs.emptyList
```

Comment:
Please remove unneeded print statements

Reply:
This should be fixed now. Can you review again and resolve the conversation if you think it's fixed.